src/Symfony/Component/HttpKernel/Client.php

```diff
@@ -50,6 +51,26 @@ public function __construct(HttpKernelInterface $kernel, array $server = array()
     }
 
     /**
+     * Sets whether to catch exceptions when the kernel is handling a request.
+     *
+     * @param bool $catchExceptions Whether to catch exceptions
+     */
+    public function catchExceptions($catchExceptions)
+    {
+        $this->catchExceptions = $catchExceptions;
+    }
+
+    /**
+     * Returns whether client is catching kernel exceptions or not.
+     *
+     * @return bool
+     */
+    public function isCatchingExceptions()
```

Comment:
Is this method really useful?

Reply:
Not really, I was just matching `Symfony\Component\BrowserKit\Client::isFollowingRedirects()`

Reply:
let's remove it :)

Reply:
Done.